test/net/sourceforge/kolmafia/textui/command/FaxbotCommandTest.java

```diff
@@ -0,0 +1,77 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import static internal.helpers.HttpClientWrapper.getRequests;
+import static internal.helpers.Networking.assertGetRequest;
+import static internal.helpers.Player.withProperty;
+
+import internal.helpers.Cleanups;
+import internal.helpers.HttpClientWrapper;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.chat.ChatManager;
+import net.sourceforge.kolmafia.persistence.FaxBotDatabase;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.session.ChoiceManager;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class FaxbotCommandTest extends AbstractCommandTestBase {
+  public FaxbotCommandTest() {
+    this.command = "faxbot";
+  }
+
+  @BeforeAll
+  public static void beforeAll() {
+    KoLCharacter.reset("testUser");
```

Comment:
There is now an unstated preference that the username in a test can be mapped to a specific file that contains one or more tests using that name.  This helps identify tests that leak or fail to clean up completely,